checkup.go

```diff
@@ -275,6 +277,15 @@ func (c *Checkup) UnmarshalJSON(b []byte) error {
 				return err
 			}
 			c.Checkers = append(c.Checkers, checker)
+		case "tcp":
+			var checker TCPChecker
+			// Default timeout
+			checker.Timeout = 1 * time.Second
```

Comment:
i actually did look in Unmarshal for a different default... what about putting it in doChecks (the "proper" place, but using a exported constant that way it shows up in the godocs prominently (or unexported and just at the top of the file...) for better discoverability.